share/da.po

```diff
@@ -131,6 +131,10 @@ msgid ""
 "Local IP address that the test engine should try to send its requests from."
 msgstr "Lokal IP-adresse som test-motoren skal sende sin foresgpørgsler fra."
 
+#, perl-brace-format
+msgid "Address {address} cannot be used as source address for DNS queries.\n"
+msgstr "Address {address} cannot be used as source address for DNS queries.\n"
+
```

Comment:
I'm sorry I didn't spot this earlier, but I don't think this should be included. It only makes it difficult for the Danish translators to notice that there is something new for them to translate.